tools/jhbuild/about.md

```diff
@@ -0,0 +1,53 @@
+---
+title: JHBuild
+subsection: jhbuild
+section: tools
+description: Build tool for GNOME development which downloads, configures and builds the code while figuring out dependencies.
+---
+
+# What is JHBuild
+[JHBuild](https://developer.gnome.org/jhbuild/stable/introduction.html.en) is the recommended way to build and run GNOME development code. It will download the code, configure and build it for you, in a way that does not interfere with or modify your existing system. JHBuild will also figure out which dependencies are required from your distribution, and it will build any development software that is needed.
+
+## Installing JHBuild on Fedora
+
+Install basic required packages
+```
+$ sudo dnf install gnome-common yelp-tools autoconf gettext-devel docbook-xsl perl-XML-Parser cvs flex bison svn glib glib-devel pygobject2 dbus-python redhat-rpm-config
+```
+
+Make directory ~/jhbuild/checkout where all the code will live and cd to it
+```
+$ mkdir ~/jhbuild/checkout
+$ cd ~/jhbuild/checkout
+```
+
+Clone jhbuild
+```
+$ git clone --depth=1 git://git.gnome.org/jhbuild
+```
+
+Cd to the source directory and build it
+```
+$ cd ~/jhbuild/
+$ ./autogen.sh
+$ make
+$ make install
```

Comment:
@meghprkh Yes, copr seems like a good way to compile this code. Not on a users machine. I find it is actually [quite easy](https://docs.pagure.org/copr.copr/user_documentation.html), and it can build on github hooks too. Please consider it